instance/models/mixins/openedx_config.py

```diff
@@ -82,7 +82,18 @@ def _get_configuration_variables(self):
                     "BLOG": "",
                     # use different contact page
                     "CONTACT": "/contact",
-                }
+                },
+                # default values
+                "MKTG_URL_LINK_MAP": {
+                    "FAQ": "help",
+                    "PRESS": "press",
+                    "SITEMAP.XML": "sitemap_xml",
+                    "COURSES": "courses",
+                    "ROOT": "root",
+                    "WHAT_IS_VERIFIED_CERT": "verified-certificate",
+                    # "BLOG": "blog",  not supported yet
+                    **self.instance.get_mktg_url_link(),
+                },
```

Comment:
@s0b0lev, @nizarmah, this breaks instances using `EDXAPP_MKTG_URL_LINK_MAP` to override the links. The proper way to override this will be to set `EDXAPP_MKTG_URL_LINK_MAP` so that instance configuration extra settings can override either by adding the `MKTG_URL_LINK_MAP` key under `EDXAPP_LMS_ENV_EXTRA` or by overriding `EDXAPP_MKTG_URL_LINK_MAP`.

Reply:
@lgp171188 thanks for bringing this up to our attention. Sorry that I wasn't aware of that issue as a reviewer. 
I can create a follow up ticket for this issue and tentatively assign it to @s0b0lev, let me know please.

For now, should we revert the change from this PR?

Reply:
@nizarmah, I have left a comment about this on the task and have already created/tentatively assigned the follow-up task. 👍🏼